package.json

```diff
@@ -1,88 +1,88 @@
 {
-    "name": "better-phpunit",
```

Comment:
I think you may have converted the spaces in package.json to tabs? Maybe just remove this file from the PR?

Reply:
Is there an easy way to remove this file from the PR? - Since it's already in a commit. IDK. Any ideas?

Reply:
I closed this one and opened a fresh PR. Easier that way. @calebporzio